src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -18,11 +18,93 @@
 from decisionengine.framework.modules.Transform import Transform
 from decisionengine.framework.taskmanager.module_graph import ensure_no_circularities
 from decisionengine.framework.taskmanager.ProcessingState import ProcessingState, State
+from decisionengine.framework.util.metrics import *
```

Comment:
Generally `import *` makes me nervous....

Can you add a `__all__` to `decisionengine.framework.util.metrics` to help limit the risk here?

Reply:
Sure thing.

Reply:
Actually, would it be better to add an __all__ there, or just import exactly what I need (which right now _is_ *, but could well change in the future)?

Reply:
Lets go with both.  Add an `__all__` and import only the specifics that you need.  Generally I find an `__all__` lets me feel better about adding local functions I don't intend to export.